scripts/qa_crowbarsetup.sh

```diff
@@ -2963,6 +2963,9 @@ function deploy_single_proposal()
                 fi
             fi
             if iscloudver 6M9plus ; then
+                # we skip all the rest of the manila setup in case of an upgrade
+                # as the images and sec-groups are already in place
+                [[ -n $upgrade_cloudsource ]] && continue
```

Comment:
>> Wouldn't it be better to check if the image is already there?

> that happens actually (with a wget option) in oncontroller_manila_generic_driver_setup.

i guess you meant checking if the image is in the nova database. anyways I think this check here at this place makes most sense, otherwise we would have to not only check if the image is there, but also check if all sec-groups are configured correctly, which (I think) we can rely on that it has been done already (as we are in an upgrade)